### DISCUSSION TITLE
(March 2021: disabled most discussions: please use "Issues")

### DISCUSSION THREAD

**Discussion by ocornut** [ocornut/imgui#3971] [Old/Archived discussions]
I opened this "Discussions" thing a few months ago but the feature currently severely lacking......

- ~~No labeling.~~ Added 2021/08
- Reference from commits not visible in threads.
- Link to other threads not visible.

As a result, I believe GitHub Discussions are currently not adequate enough.

**In the meantime, I encourage you to open "Issues" instead of discussions.** 

Discussions always traditionally happened in Issues, and until GitHub Discussions becomes a useful tool for long-term sharing/searching of information I believe it shouldn't be used. If GitHub don't fix Discussions I would consider just closing this alltogether but I am waiting for a discussions>issue conversion path.

**Comment by mathisloge** [comment]
* labeling feature tracking: https://github.com/github/feedback/discussions/19
* commits https://github.com/github/feedback/discussions/3562
* mark as implemented etc.  https://github.com/github/feedback/discussions/3192

**Reply by mathisloge** [reply]
issue to discussion rolled out in public beta : https://github.com/github/feedback/discussions/2952 (https://github.com/github/feedback/discussions/2952#discussioncomment-328878)

**Reply by mathisloge** [reply]
Discussions are now officially out of beta. https://github.blog/2021-08-17-github-discussions-out-of-beta/

**Comment by rav-en** [comment]
Hi, im sure im doing something wrong here, forgive my incompetence. Just trying to figure out the demo before adding it into my current project. Getting the current error when compiling fatal error LNK1120 when running imgui_demo.cpp

Also tried some examples, but had an errors of not being able to find files when running the .bat file
fatal error C1083: Cannot open include file: 'float.h': No such file or directory
glcorearb.h(60): fatal error C1083: Cannot open include file: 'windows.h': No such file or directory

ive tried playing around with it for an hour or 2 and cant seem to get past these issues.
Would love some help.

Kind regards.

**Reply by PathogenDavid** [reply]
> Also tried some examples, but had an errors of not being able to find files when running the .bat file

Unless you have a compelling reason to use the batch scripts, it's generally easier to just open the `examples/imgui_examples.sln` solution file in Visual Studio.

If you insist on using the batch scripts, you need to run them from a Visual Studio developer command prompt. (If you search for Developer Command Prompt in Windows it should come up.)

**Reply by rav-en** [reply]
Thank you! i didnt scroll down all the way to imgui_examples.sln. 👍

**Comment by PathogenDavid** [comment]
@ocornut

FYI, [the issue template still refers to using GitHub discussions](https://github.com/ocornut/imgui/blame/8877eab3933c7031e2b41f372b852618db219489/.github/issue_template.md#L7) for linker/compilation/etc issues.

Also GitHub [announced labels for GitHub discussions the other day](https://github.blog/changelog/2021-05-18-github-discussions-labels-and-announcements-category-format/). I haven't had a chance to mess with them yet, but hopefully that particular issue is solved now.

**Comment by YusufSuleman** [comment]
@ocornut, Can we still post things about ImGui, that are not wanting help, but talk about ImGui.

**Reply by ocornut** [reply]
I am going to reinvestigate the feature set of Discussions soon and see how we could use it again.

**Comment by ocornut** [comment]
I have reevaluated GitHub discussions today.
It is a little better (still outside references not showing in a topic... hurts discoverability a lot) but I think it wouldn't be helpful to move too many topics here.

I'm going to start reopening with a **single category dedicated to people posting their "Getting Started" Link/Build/Integration issues**: #4497

In the future we might add other categories but I believe gathering things around Issues is generally OK.

**Reply by YusufSuleman** [reply]
I heard that Disscudsions is out of beta, a few days a ago.
